client/src/components/FlashCards.js

```diff
@@ -4,44 +4,6 @@ import { useDrag } from 'react-use-gesture'
 import Card from './Card'
 import '../css/FlashCards.css'
 
-const cardData = [
-  {
-    title:'Did you know?',
-    body:'This virus was first detected in Wuhan City, Hubei Province, China. The first infections were linked to a live animal market, but the virus is now spreading from person-to-person.',
-    img:'https://upload.wikimedia.org/wikipedia/en/f/f5/RWS_Tarot_08_Strength.jpg',
-    footer:'fun fact# 1'
-  },
-  {
-    title:'Did you know?',
-    body:'The virus causing coronavirus disease 2019 (COVID-19), is not the same as the coronaviruses that commonly circulate among humans and cause mild illness, like the common cold.',
-    img:'https://upload.wikimedia.org/wikipedia/en/5/53/RWS_Tarot_16_Tower.jpg',
-    footer:'fun fact# 2'
-  },
-  {
-    title:'1',
-    body:'5',
-    img:'https://upload.wikimedia.org/wikipedia/en/9/9b/RWS_Tarot_07_Chariot.jpg',
-    footer:'fun fact# 3'
-  },
-  {
-    title:'1',
-    body:'4',
-    img:'https://upload.wikimedia.org/wikipedia/en/thumb/8/88/RWS_Tarot_02_High_Priestess.jpg/690px-RWS_Tarot_02_High_Priestess.jpg',
-    footer:'fun fact# 4'
-  },
-  {
-    title:'1',
-    body:'3',
-    img:'https://upload.wikimedia.org/wikipedia/en/d/de/RWS_Tarot_01_Magician.jpg',
-    footer:'fun fact# 5'
-  },
-  {
-    title:'Did you know?',
-    body:"In Covid-19,'CO' stands for 'corona', 'VI' for 'virus', and 'D' for disease. Formerly, this disease was referred to as 2019 novel coronavirus or 2019-nCov. There are many types of human coronaviruses including some that commonly cause mild upper-respiratory tract illness",
-    footer:'fun fact# 6'
-  }
-]
-
```

Comment:
this was meant to move into `fakeCardData.js` right?

Reply:
yea i moved it just to make it easier to code out the component

Reply:
cool cool